common/scripts/upgrade.sh

```diff
@@ -410,6 +411,29 @@ __cleanup_workers() {
   fi
 }
 
+__upgrade_local_system_node_components() {
+  __process_marker "upgrading local system nodes"
+  __get_systemClusters
+  if [ "$(echo $system_cluster | jq -r '.')" != null ]; then
+    sudo ./manageSharedNode clean
+    sudo ./manageSharedNode add
+  fi
+}
+
+__get_systemClusters() {
```

Comment:
We should probably name this something like `get_default_systemCluster`. The current name suggests that we are getting all `systemClusters`.